### DISCUSSION TITLE
How could I use disroot NextCloud from LXQt?

### DISCUSSION THREAD

**Discussion by rajibando** [lxqt/lxqt#2643] [Q&A]
Guidelines for Gnome and KDE are given. But `Online Accounts` service, though appears to be in KDE and Gnome, doesn't appear in LXQt. The program in synaptic appears only to be for `gnome-online-accounts`.

Upon my questioning the Support Team, Disroot Cloud, I was told that for collaborative work, perhaps it's better to use something more suitable like Nextcloud, which allows for document editing (also commenting, chatting and videocalls in one) additionally, we are about to enable this app: https://apps.nextcloud.com/apps/collectives

Also I was told that I might be interested in https://cryptpad.disroot.org its set of collaboration tools similar to etherpad or nextcloud, but those are end to end encrypted on top of it. One doesn't need to create account to use it, but if one does, one can create teams and group all documents (one needs to create separate account for it, which also means one doesn't need to wait for approval).

Information material:
[GNU/Linux | Howto Disroot](https://howto.disroot.org/en/tutorials/cloud/clients/ondesktop/gnu-linux)
Linux desktops integrate with nextcloud very well. The tutorials be checked:
https://howto.disroot.org/en/tutorials/cloud/clients/ondesktop/gnu-linux/kde
https://howto.disroot.org/en/tutorials/cloud/clients/ondesktop/gnu-linux/gnome

`nextcloud disk` could also manually be connected using `webdav`. Generally the url should be: https://cloud.disroot.org/remote.php/dav/files/USERNAME/. The following little tutorial about adding mounts to various file browsers please be checked:
[Accessing Nextcloud files using WebDAV](https://github.com/nextcloud/documentation/blob/master/user_manual/files/access_webdav.rst)

**Comment by tsujan** [comment]
> ... from LXDe

Not an answer to your question — contrary to https://github.com/lxqt/lxqt/discussions/2642, I have no experience with NextCloud — but you probably meant *LXQt*, not LXDE.

Hopefully, someone who knows about NextCloud will help you.

**Reply by rajibando** [reply]
Regarding
> ... from LXDe

and
> you probably meant LXQt, not LXDE

Apologies, Mr. Tsujan! Old memory resurfaced to tint my flow of thoughts and typing, and altered my choice of the defining term. Have corrected the error.

> Hopefully, someone who knows about NextCloud will help you.

Ok, will then be compelled to wait.  But may be the tools , `Online Accounts` and `webdav` services, could be the requirement.
